File: JULIA/Cap03/c03ex06.jl
# c03ex06.jl

palavra = "COMPUTADOR"

println("Palavra ........: $palavra")
println()
println("Lado esquerdo ..: $(palavra[1:3])")
println("Centro .........: $(palavra[4:7])")
println("Lado direito ...: $(palavra[end-2:end])")
println("Reverso ........: $(reverse(palavra))")
println("Copias .........: $(palavra ^ 3)")
println("Tira pedaco ....: $(palavra[1:7])")
println("Nova palavra ...: $(palavra[1:6])O$(palavra[8:end])")

print("\nPressione <Enter> para encerrar...")
readline()

File: SCALA/Cap03/c03ex06.scala
// c03ex06.scala

import scala.io.StdIn._

object Main {
  def main(args: Array[String]): Unit = {
    val palavra = "COMPUTADOR"
    val tamanho = palavra.length
    val copias = palavra + palavra + palavra
    val reverso = new StringBuilder(palavra).reverse.toString
    val nvPalavra = palavra.substring(0, 6) + "O" + palavra.substring(7)

    println("Palavra ........: " + palavra)
    println()

    println("Lado esquerdo ..: " + palavra.substring(0, 3))
    println("Centro .........: " + palavra.substring(3, 7))
    print("Lado direito ...: ")
    println(palavra.substring(tamanho - 3))
    println("Reverso ........: " + reverso)
    println("Copias .........: " + copias)
    println("Tira pedaco ....: " + palavra.substring(0, 7))
    println("Nova palavra ...: " + nvPalavra)

    print("\nTecle <Enter> para encerrar... ")
    readLine()
  }
}
